src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/prisonersearch/PrisonerSearchApiClient.kt

```diff
@@ -39,10 +39,10 @@ class PrisonerSearchApiClient(@Qualifier("prisonerSearchApiWebClient") private v
       ?: emptyList()
   }
 
-  fun matchPrisonerByNameAndDateOfBirth(searchByNameAndDateOfBirthOrPncNumber: SearchByNameAndDateOfBirth): List<Prisoner> {
+  fun matchPrisonerByName(searchByLastName: SearchByLastName): List<Prisoner> {
```

Comment:
I think a name that makes it clear that we maybe matching by name and date of birth would be better?